dynawo/sources/Common/DYNTrace.cpp

```diff
@@ -147,22 +147,23 @@ void Trace::addAppenders_(const std::vector<TraceAppender>& appenders) {
   // remove old appenders (console_log)
   Trace::resetCustomAppenders();
 
+  traceAppenders_.clear();  // clear previous appenders for a new job
```

Comment:
Maybe we should create a clearAppenders method to do this? Otherwise it means we cannot call addAppenders twice in a row. Or maybe rename the method